go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/antonmedv/expr v1.15.5
-	github.com/argoproj/notifications-engine v0.4.1-0.20231213155711-c02dc5fdb5e1
+	github.com/argoproj/notifications-engine v0.4.1-0.20240219110818-7a069766e954
```

Comment:
Updating to https://github.com/argoproj/notifications-engine/commit/7a069766e95476e1074eeed6145085160e2fec63